### ISSUE TITLE
Isolate single language processing library

### ISSUE THREAD

**Issue by msarahan** [ContinuumIO/topik#14]
- Gensim
 - NLTK
 - 

Reimplement internally

**Comment by msarahan** [comment]
Current usages:

Gensim: LDA (was used also for stopwords, but we don't really need that)
NLTK: collocation tokenizer
TextBlob: entities tokenizer (depends in turn on NLTK)

**Comment by gpfreitas** [comment]
@msarahan , did you guys consider [spacy](https://spacy.io/)? If so, any thoughts on it?

**Comment by msarahan** [comment]
No, we didn't. Sorry, other than being aware of it by name, I don't know anything about it.

**Comment by gpfreitas** [comment]
No worries. I am trying to find out more about this library. Very few people seem to know it well. Thanks, Mike.
